internal/oci/uvm_test.go

```diff
@@ -1,12 +1,283 @@
 package oci
 
 import (
+	"context"
 	"testing"
 
 	runhcsopts "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/options"
 	"github.com/opencontainers/runtime-spec/specs-go"
 )
 
+func Test_ParseAnnotationsCPULimit(t *testing.T) {
+	type tc struct {
+		name string
+
```

Comment:
What's with these blank lines?

Reply:
No reason. Seemed like they were logically different areas of the struct so I did.